monitoring/dashboards/viya/sas-launched-jobs-node.json

```diff
@@ -156,9 +172,12 @@
         "text": {},
         "textMode": "value_and_name"
       },
-      "pluginVersion": "8.2.1",
+      "pluginVersion": "9.0.3",
       "targets": [
```

Comment:
See comment above re: Grafana 9.x vs. Grafana 8.x

Reply:
I just tested the pluginVersion on Grafana 8.4 and all of the dashboards still worked.